util/pom.xml

```diff
@@ -73,18 +73,6 @@
 
     <build>
         <plugins>
-            <plugin>
```

Comment:
removed these since there is no change from parent

Reply:
I remember I had some issue with things not working unless I included it, and so I added it...

Reply:
Can we add this back?

With this change it seems the following don't seem to be included when I run `mvn compile`
```
[INFO] --- protoc-jar-maven-plugin:3.11.4:run (default) @ datacommons-import-util ---

[INFO] --- fmt-maven-plugin:2.11:format (default) @ datacommons-import-util ---
```